src/js/player.js

```diff
@@ -300,7 +300,17 @@ class Player extends Component {
     if (tag.parentNode) {
       tag.parentNode.insertBefore(el, tag);
     }
-    Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    //Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    // append the DOM el function to make use of our component.addChild method
+    // to properly be injected into the children Array
+    tag.el = function () {
```

Comment:
I think we should update it to be something like:
```js
// insert the tag as the first child of the player element
// then manually add it to the children array so that this.addChild 
// will work properly for other components
Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
this.children_.unshift(tag);
```